front/app/containers/Admin/communityMonitor/components/LiveMonitor/components/HealthScoreWidget/messages.ts

```diff
@@ -0,0 +1,24 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  noResults: {
+    id: 'app.containers.Admin.communityMonitor.noResults',
+    defaultMessage: 'No results for this period.',
+  },
+  quarterChartLabel: {
+    id: 'app.containers.Admin.communityMonitor.quarterChartLabel',
+    defaultMessage: 'Q{quarter} {year}',
+  },
+  healthScore: {
+    id: 'app.containers.Admin.communityMonitor.healthScore',
+    defaultMessage: 'Health Score',
+  },
+  value: {
+    id: 'app.containers.Admin.communityMonitor.value',
+    defaultMessage: '{value}',
+  },
```

Comment:
Wasn't quite sure why this would need a translation string?

Reply:
Whoops - an unused message :sweat_smile: Will remove.